app/src/main/java/org/wikipedia/readinglist/LongPressMenu.kt

```diff
@@ -85,10 +85,15 @@ class LongPressMenu(private val anchorView: View, private val existsInAnyList: B
 
     private fun deleteOrShowDialog() {
         listsContainingPage?.let { list ->
-            RemoveFromReadingListsDialog(list).deleteOrShowDialog(anchorView.context) { _: List<ReadingList?>?, _: ReadingListPage? ->
-                entry?.let {
+            RemoveFromReadingListsDialog(list).deleteOrShowDialog(anchorView.context) { readingLists: List<ReadingList?>, _: ReadingListPage ->
+                entry?.let { it ->
                     if (anchorView.isAttachedToWindow) {
-                        showMessage((anchorView.context as AppCompatActivity), anchorView.context.getString(R.string.reading_list_item_deleted, it.title.displayText))
+                        val readingListNames = mutableListOf<String>()
+                        readingLists.forEach { readingList ->
+                            readingListNames.add(readingList!!.title())
+                        }
+                        showMessage((anchorView.context as AppCompatActivity), anchorView.context.getString(R.string.reading_list_item_deleted_from_list,
+                                it.title.displayText, readingListNames.joinToString(separator = ", ")))
```

Comment:
Something to consider: Not all languages use the comma `,` to separate items in a list. How will this look in Arabic?

The real proper solution will be to use the ICU `ListFormatter` class, but this can be done in a future update.